releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -103,44 +109,77 @@ public static Result buildResult(String localRepoPath, String authToken, String
                 System.out.println(commitMessage.getMessage());
                 commitMessage = new CommitMessage(commitMessage.getRevertedCommitMessage());
             }
-            if (commitMessage.isIssueOrPull()) {
-                final int issueNo = commitMessage.getIssueNumber();
-                if (processedIssueNumbers.contains(issueNo)) {
-                    continue;
-                }
-                processedIssueNumbers.add(issueNo);
+            buildResultWithLabel(remoteRepoPath, result, remoteRepo, commitsForRelease,
```

Comment:
make it like:
`result = buildResultWithLabel(.......)`
`new Result()` should be inside `buildResultWithLabel`

Reply:
Changes Done

Reply:
I believe it will give an error like If I do:

```
   final Result result =  buildResultWithLabel(remoteRepoPath, new Result(), remoteRepo, commitsForRelease,
                                 commit, commitMessage);
```

Then I have to make another return which will lead to have 2 return statements in a method which will give me checkstyle violation error.

Reply:
put `new Result()` inside method. no need to put it in paramerter.

name method `createResult`

Reply:
Changes Done